src/coreclr/vm/fcall.h

```diff
@@ -582,8 +582,9 @@ LPVOID __FCThrowArgument(LPVOID me, enum RuntimeExceptionKind reKind, LPCWSTR ar
         HELPER_METHOD_FRAME_BEGIN_EX_BODY(ret, helperFrame, gcpoll, allowGC)    \
             /* <TODO>TODO TURN THIS ON!!!   </TODO> */                    \
             /* gcpoll; */                                                       \
+            if (g_isNewExceptionHandlingEnabled) __helperframe.Push();         \
```

Comment:
This has risk of introducing regressions in assorted set of micro-benchmarks.

Reply:
I have ran a subset of the microbenchmarks and this was not causing any visible effect.